pyanaconda/payload/dnf/utils.py

```diff
@@ -32,16 +30,9 @@
 
 DNF_PACKAGE_CACHE_DIR_SUFFIX = 'dnf.package.cache'
 YUM_REPOS_DIR = "/etc/yum.repos.d/"
-
 USER_AGENT = "%s (anaconda)/%s" % (productName, productVersion)
 
 
-def go_to_failure_limbo():
```

Comment:
Wouldn't this change just kill the Anaconda with a traceback instead of showing GUI error to user?

Reply:
The traceback will be processed in the installation task (from `pyanaconda.installation_tasks`).